tsl/test/sql/dist_hypertable_pg12.sql

```diff
@@ -29,7 +30,7 @@ CREATE ACCESS METHOD test_am TYPE TABLE HANDLER heap_tableam_handler;
 $$);
 
 -- Create distributed hypertable using non-default access method
-CREATE TABLE disttable(time timestamptz NOT NULL, device int) USING test_am;
+CREATE TABLE disttable(time timestamptz NOT NULL, device int, temp_c float, temp_f float GENERATED ALWAYS AS (temp_c * 9 / 5 + 32) STORED) USING test_am;
```

Comment:
I don't see the connection between `test_am` and testing `GENERATED` columns support. Is there?

Reply:
No connection. Just reusing existing table instead of adding a new one.

This test file previously contained only tests for AM, as this was a new feature in PG12, thus can't run on PG11. I figured it is better to put all tests that require PG12 or higher in the same test file as the test cannot go into `dist_hypertable`.